docs/ide/quickstart-nodejs.md

```diff
@@ -15,94 +15,115 @@ dev_langs:
-    From the top menu bar, choose **File** > **New** > **Project**. In the left pane of the **New Project** dialog box, expand **JavaScript**, then choose **Node.js**. In the middle pane, choose **Blank Node.js Web application**, then choose **OK**.
+    1. From the top menu bar, choose **File > New > Project**.
+
+    1. In the left pane of the **New Project** dialog, expand **JavaScript** and choose **Node.js**.
+
+    1. In the middle pane, choose **Blank Node.js Web application** and select **OK**.
+
     ::: moniker-end
-    If you don't see the **Blank Node.js Web application** project template, you must add the **Node.js development** workload. For detailed instructions, see the [Prerequisites](#prerequisites).
+    
+    If you don't see the **Blank Node.js Web application** project template, you must add the **Node.js development** workload. (For detailed instructions, see the [prerequisites](#prerequisites).)
 
-    Visual Studio creates and the new solution and opens the project. *server.js* opens in the editor in the left pane.
+    Visual Studio creates and opens the project. The project's *server.js* file opens in the editor on the left.
 
 ## Explore the IDE
 
-1. Take a look at **Solution Explorer** in the right pane.
+1. Look at the **Solution Explorer** in the right pane.
 
    ![Solution Explorer](../ide/media/quickstart-nodejs-solution-explorer.png)
 
-   - Highlighted in bold is your project, using the name you gave in the **New Project** dialog box. On disk, this project is represented by a *.njsproj* file in your project folder.
+   - Highlighted in bold is your project, using the name provided when you set up the project. On disk, this project is represented by a *.njsproj* file in your project folder.
 
    - At the top level is a solution, which by default has the same name as your project. A solution, represented by a *.sln* file on disk, is a container for one or more related projects.
 
-   - The npm node shows any installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
+   - The **npm** node shows installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
 
 1. If you want to install npm packages or Node.js commands from a command prompt, right-click the project node and choose **Open Command Prompt Here**.
 
    ![Node.js command prompt](../ide/media/quickstart-nodejs-command-prompt.png)
 
-1. In the *server.js* file in the editor (left pane), choose `http.createServer` and then press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
+1. In the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
```

Comment:
I could be missing something, but when I went through the quickstart, the `createServer` function was in my *http.d.ts* file and not an *index.d.ts* file. Also, should we explain what this step is for? It isn’t clear to me.